packages/contracts/contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -453,25 +453,27 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
 
         // We will sequentially append leaves which are pointers to the queue.
         // The initial queue index is what is currently in storage.
-        uint40 nextQueueIndex = getNextQueueIndex();
+        uint40 firstSubsequentQueueIndex = getNextQueueIndex();
```

Comment:
this is a bit of a clunky name, but it's helpful, because otherwise it suggests a too-strong relationship between `nextQueueIndex` and `nextContext` as used below.

Reply:
I think it's worth considering calling this `subsequentQueueIndex` or `curSubsequentQueueIndex` since we update it in the loop below.  Feels a little confusing for it to be called the "first" in that context.  Thoughts?  I guess the counter would be it is always the first subsequent index for whatever context we are on.